bundles/org.openhab.binding.keba/README.md

```diff
@@ -4,36 +4,45 @@ This binding integrates the [Keba KeContact EV Charging Stations](http://www.keb
 
 ## Supported Things
 
-Currently the Keba KeContact P20 and P30 stations are supported by this binding.
+The Keba KeContact P20 and P30 stations are supported by this binding, the thing type id is `kecontact`.
 
-## Binding Configuration
-
-The binding uses the default UDP port number to connect to the Keba Charging Station.
-So, no special configuration of the binding itself is required.
 
 ## Thing Configuration
 
-The Keba KeContact P20/30 requires the ip address as a configuration value in order for the binding to know where to access it. Optionally, a refresh interval (in seconds) can be defined that steers the polling of the actual state of the charging station
-
-In the thing file, this looks e.g. like
-
-```
-Thing keba:kecontact:1 [ ipAddress="192.168.0.64", refreshInterval=30 ]
-```
+The Keba KeContact P20/30 requires the ip address as the configuration parameter `ipAddress`. Optionally, a refresh interval (in seconds) can be defined as parameter `refreshInterval` that defines the polling of values from the charging station.
 
-the refreshInterval can optionally be used in combination with ```autoupdate="false"``` in the .items file so that the Items are updated with the latest actual value sent by the charging station.
 
 ## Channels
 
 All devices support the following channels (non exhaustive):
 
-| Channel Type ID  | Item Type | Description                                                                                       |   |   |
-|------------------|-----------|---------------------------------------------------------------------------------------------------|---|---|
-| state            | Number    | This channel indicates the current operational state of the wallbox                               |   |   |
-| maxpresetcurrent | Number    | This channel supports adjusting the maximum current the charging station should deliver to the EV |   |   |
-| power            | Number    | This channel indicates the active power delivered by the charging station                         |   |   |
-| I1/2/3           | Number    | This channel indicates the current for the given phase                                            |   |   |
-| U1/2/3           | Number    | This channel indicates the voltage for the given phase                                            |   |   |
+| Channel ID         | Item Type | Read-only | Description                                                            |   |   |
+|--------------------|-----------|------------------------------------------------------------------------------------|---|---|
```

Comment:
The table is no longer properly rendered. I think it can be fixed with this change:

```suggestion
|--------------------|-----------|-----------|------------------------------------------------------------------------|---|---|
```

While you're at it, can you also remove the 2 right most unused columns?